dts/bindings/radio/radio.yaml

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
I guess this binding would not fit for SPI transceivers, right? Would it be possible to rename it to radio-memio.yaml?

Reply:
what mean memio? Do you mean that it internally maped, embedded soc radio?

Reply:
yes

Reply:
changed to: soc-radio